xbmc/storage/discs/DiscDriveHandlerBase.h

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "IDiscDriveHandler.h"
+
+#include <string>
+
+/*! \brief Implementation of a generic/base DiscDriveHandler
+* \sa IDiscDriveHandler
+*/
+class CDiscDriveHandlerBase : public IDiscDriveHandler
+{
+public:
+  /*! \brief Toggle the state of a given drive device
```

Comment:
```c++
  CDiscDriveHandlerBase() = default;
  ~CDiscDriveHandlerBase() override = default;
```

Reply:
Just curious why this change is needed, is it some sort of best practice to always define the default constructor/dtor? I would assume they'd be automatically defined to default if not explicitly defined

Reply:
i also scratched my head about this a hundred times.. this sounds like a good explanation to me https://stackoverflow.com/questions/20828907/the-new-syntax-default-in-c11 (istrivial / ispod) 🤔

Reply:
https://en.wikipedia.org/wiki/Rule_of_three_(C%2B%2B_programming)